Rubrik/Public/Get-RubrikVcdTemplateExportOptions.ps1

```diff
@@ -0,0 +1,96 @@
+#requires -Version 3
+function Get-RubrikVcdTemplateExportOptions
```

Comment:
Is it intentional that you use `Vcd` when referencing templates but `VCD` when referencing the server?

Reply:
Generally I try to use vCD in documentation or anything that is not actual code. Within CDM `Vcd` is typically used in object IDs and other places. Cmdlet names typically use UpperCamelCase, so I will do a quick pass and make sure there is consistency, at very least.